tests/integration/pure_cairo_vm/cheatcodes/call/call_test.cairo

```diff
@@ -0,0 +1,18 @@
+from starkware.cairo.common.math import assert_not_zero
+
+func test_call(){
+    alloc_locals;
+    local deployed_contract_address;
+
+    %{ ids.deployed_contract_address = deploy_contract("./src/basic.cairo").contract_address %}
+    assert_not_zero(deployed_contract_address);
+
+    %{
+        result = call(ids.deployed_contract_address, "get_balance")
+        assert result == [100]
+        call(ids.deployed_contract_address, "increase_balance", [50])
```

Comment:
this should not modify the state

Reply:
That's the point of this test, first, we check if the balance is `100`, then we increase it by `50` and then again we expect it to be `100` to ensure that the state is not modified.

Reply:
ah right my bad🤦